### ISSUE TITLE
1.0.0-rc-1 Duplicate declaration "typeOfOriginalExport"

### ISSUE THREAD

**Issue by TheSavior** [speedskater/babel-plugin-rewire#106]
With 1.0.0-rc-1, I can get the following error:

```
> browserify -t babelify test.js > test.bundle.js && node test.bundle.js

TypeError: babel-test-case/test_module/index.js: Duplicate declaration "typeOfOriginalExport" (This is an error on an internal node. Probably an internal error)
    at File.buildCodeFrameError (babel-test-case/node_modules/babel-core/lib/transformation/file/index.js:407:15)
    at Scope.checkBlockScopedCollisions (babel-test-case/node_modules/babel-core/node_modules/babel-traverse/lib/scope/index.js:433:27)
    at Scope.<anonymous> (babel-test-case/node_modules/babel-core/node_modules/babel-traverse/lib/scope/index.js:574:18)
    at Scope.registerBinding (babel-test-case/node_modules/babel-core/node_modules/babel-traverse/lib/scope/index.js:588:8)
    at Scope.registerDeclaration (babel-test-case/node_modules/babel-core/node_modules/babel-traverse/lib/scope/index.js:509:14)
    at Object.BlockScoped (babel-test-case/node_modules/babel-core/node_modules/babel-traverse/lib/scope/index.js:198:28)
    at Object.newFn (babel-test-case/node_modules/babel-core/node_modules/babel-traverse/lib/visitors.js:295:17)
    at NodePath._call (babel-test-case/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:63:18)
    at NodePath.call (babel-test-case/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:43:14)
    at NodePath.visit (babel-test-case/node_modules/babel-core/node_modules/babel-traverse/lib/path/context.js:93:12)
    at TraversalContext.visitQueue (babel-test-case/node_modules/babel-core/node_modules/babel-traverse/lib/context.js:146:16)
```

I have been able to create a minimal test case here: https://github.com/TheSavior/babel-plugin-rewire-test-case

I'm not sure how to make a test case in this repo since the test suite uses a global hook to transform and thus not sure how to specify a babel rc for the nested package.

The problem occurs when requiring with browserify a package that also uses this plugin.

**Comment by speedskater** [comment]
@TheSavior Thanks for providing the error report. I will have to look at the example in more detail to analyze the error and integrate it into our samples tests.
I will try to tackle the issue on the weekend.

**Comment by StyMaar** [comment]
> The problem occurs when requiring with browserify a package that also uses this plugin. 
It also occurs with no browserify at all in my case.

**Comment by StyMaar** [comment]
My bad, in my case it was caused by two `export default` statements in a row. Interestingly enough, this mistake caused no error without the plugin.

**Comment by speedskater** [comment]
@TheSavior, @StyMaar The fix is incorporated in 1.0.0-rc-2. Please let me know if this works for you.
